src/kakarot/evm.cairo

```diff
@@ -700,6 +711,13 @@ namespace EVM {
     ) -> Summary* {
         alloc_locals;
 
+        // Compute intrinsic gas usage
+        // See https://www.evm.codes/about#gascosts
+        let count = Helpers.count_nonzeroes(nonzeroes=0, idx=0, arr_len=calldata_len, arr=calldata);
```

Comment:
why do we need to provide a nonzeroes argument to count_nonzeroes?
i think the function pre-existed this PR but can be good to simply erase this param?